homeassistant/components/huumtest/__init__.py

```diff
@@ -0,0 +1,34 @@
+"""The A test integration for Huum Saunas integration."""
```

Comment:
Please remove all the huumtest files

Reply:
Ah, forgot to push the change, sorry, will fix